service/user.service.js

```diff
@@ -72,11 +74,17 @@ const getAllUsers = async () => {
 
 const getUser = async (email) => {
     const user = await UserModel.findOne({ email });
-    return new UserDTO(user);
+    const userData = new UserDTO(user);
+    const file = await fs.readFileSync(path.resolve(__dirname, "../uploads/avatars", user.avatar));
+    const avatar = "data:image/png;base64," + file.toString("base64");
```

Comment:
@ikadushkina why do you send image as base64?? It will increase the response body significantly and introduce some complications. I think you should make you `uploads/avatars` folder static and return the image URL instead.